easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-intel-2019b.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.1.2'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = "TopHat is a fast splice junction mapper for RNA-Seq reads."
+
+toolchain = {'name': 'intel', 'version': '2019b'}
```

Comment:
Much like the foss version, this could be dropped to `iimpi` level.

Reply:
@hcartiaux same here, move to iimpi?
```
toolchain = {'name': 'iimpi', 'version': '2019b'}
```